### PR TITLE
EDRD Sprint 6 Development

### PR DESCRIPTION
**Pull request by deepakmulamalla**
Development Related to:
EDRD-764​ | EDRD Reference Number​​
EDRD-731​ | Eligibility Form and EDRD Eligibility Form Signed Date validation that is run at Save in the AC Recommendation needs to be removed​​
EDRD-776​ | EDRD Portal Search Enhancement​​
EDRD-772​ | As an EDRD Super User I need additional permissions to enable me to access Reports and Dashboards and other Setup features​​
EDRD-749​ | Provider Portal - Select Dosage is not mandatory and request can be submitted with a blank form​
EDRD-728​ | Provider Portal - Medication Request - Clicking on Medication - Clicking on Manufacturer - Invalid Page​
EDRD-724​ | Provider Portal - In My Medication Requests - Open & Closed the provider cannot search or filter in the list views​
EDRD-769​ | Provider Portal - Renewal Submissions - Patient Related Cases - Entering Case Number Issues​
EDRD-608​ | Need to resolve the queue issue whereby EDRD users can see and access SAT queues​
EDRD-814​ | Case Owner can be changed to anyone with an active license in the org; i.e., EDRD cases can be assigned to SA users

### CODE REVIEW

File: force-app/main/default/triggers/CaseTrigger.trigger-meta.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
-<?xml version='1.0' encoding='UTF-8'?>
+<?xml version="1.0" encoding="UTF-8"?>
 <ApexTrigger xmlns="http://soap.sforce.com/2006/04/metadata">
-  <apiVersion>60.0</apiVersion>
-  <status>Active</status>
-</ApexTrigger>
\ No newline at end of file
+    <apiVersion>57.0</apiVersion>
```
Comment (cgijeffolsen) [inline force-app/main/default/triggers/CaseTrigger.trigger-meta.xml:3]: @deepakmulamalla Please advise what created the requirement for this api rollback.

Comment (deepakmulamalla) [reply force-app/main/default/triggers/CaseTrigger.trigger-meta.xml]: @cgijeffolsen, I've reverted the changes back to Api version v60.

**Review by cgijeffolsen — APPROVED** [review]
Looking good, thank you @deepakmulamalla
@yashrohan-cgi and @RanadheerRG will review before we promote the PR.
cc @NataliaNikishina

**Comment by deepakmulamalla** [comment]
> Looking good, thank you @deepakmulamalla @yashrohan-cgi and @RanadheerRG will review before we promote the PR. cc @NataliaNikishina

Thanks for approving @cgijeffolsen

**Comment by deepakmulamalla** [comment]
@yashrohan-cgi & @RanadheerRG , Is there any updates on this PR?

**Review by RanadheerRG — APPROVED** [review]
Approved @deepakmulamalla 
Cc: @cgijeffolsen

**Comment by deepakmulamalla** [comment]
@yashrohan-cgi, any updates on the PR review?

**Comment by cgijeffolsen** [comment]
@deepakmulamalla We will check in with Yash. Please proceed with the needed merge.
I have made Yash's review optional and will advise you of any items that Yash brings to me separately.

cc @NataliaNikishina @yashrohan-cgi @RanadheerRG 

Thank you,
Jeff

**Comment by deepakmulamalla** [comment]
> @deepakmulamalla We will check in with Yash. Please proceed with the needed merge. I have made Yash's review optional and will advise you of any items that Yash brings to me separately.
> 
> cc @NataliaNikishina @yashrohan-cgi @RanadheerRG
> 
> Thank you, Jeff

Thanks Jeff

File: force-app/main/default/lwc/eDRD_LC_ShowToastMessageOnCase/eDRD_LC_ShowToastMessageOnCase.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+                if (!isAttachmentPresentResult && !efsDateValue) {
+                    this.showToast(); // Warning when both conditions are not met
+                } else if (!efsDateValue) {
+                    this.showToastEDMsg(); // Warning when EFS date is missing
+                } else if (!isAttachmentPresentResult) {
+                    this.showToastEFMsg(); // Warning when attachment is missing
+                }
+            })
+            .catch(error => {
+                console.log(error);
```
Comment (yashrohan-cgi) [inline force-app/main/default/lwc/eDRD_LC_ShowToastMessageOnCase/eDRD_LC_ShowToastMessageOnCase.js:53]: remove console.log

**Pull request merged**